docs/mint.json

```diff
@@ -1412,4 +1413,4 @@
     "v3",
     "v2"
   ]
-}
\ No newline at end of file
+}
```

Comment:
```suggestion
}

```

Reply:
I like the proposed order. Looks like pre-commit isn't passing: https://github.com/PrefectHQ/prefect/actions/runs/12634420207/job/35202113415?pr=16622#step:8:73

Adding this empty new line should fix that.